GpsDataAnalyzer/test/test_csv.csv

```diff
@@ -0,0 +1,14 @@
+
+simulation_type,file_path,run_duration,gain,start_time,end_time
+DynamicSimulation,/home/ameles/Desktop/GPS-simulator/geobeam-dev/geobeam/user_motion_files/userwalking.csv,,-2,2020-07-22T15:49:01.626803,2020-07-22T15:49:20.144408
```

Comment:
run_duration is missing. Is that intended?

Reply:
yeah, if there wasn't one specified it won't end early so it will just run to completion